qvisor/src/arch/aarch64/vcpu/kvm_vcpu.rs

```diff
@@ -0,0 +1,162 @@
+// Copyright (c) 2021 Quark Container Authors / 2018 The gVisor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#![allow(non_upper_case_globals)]
+use kvm_bindings::kvm_vcpu_events;
+use crate::{KVMVcpu, qlib::common::Error, vmspace::VMSpace, qlib::backtracer};
+
+pub const _KVM_ARM_VCPU_PSCI_0_2: u32 = 2;
```

Comment:
btw ... why did we name this register encoding with a leading underscore in the first place, when they are `pub` ?

Reply:
If I understand correctly a name prefixed with `_` means "this is not meant to be called/used", or "this is only used internally. In the later case these items should not be `pub` either. This feels like a relic from early development. It's a chance to fix them here. (same may goes for some other functions).